bedrock/mozorg/templates/mozorg/about/policy/transparency/index.html

```diff
@@ -4,32 +4,32 @@
 
 {% extends "base-protocol.html" %}
 
-{% block page_title %}{{ _('Transparency') }}{% endblock %}
+{% block page_title %}Transparency{% endblock %}
 
 {% block page_css %}
-{{ css_bundle('about-transparency') }}
+  {{ css_bundle('about-transparency') }}
 {% endblock %}
 
 {% block js %}
-{{ js_bundle('about-transparency') }}
+  {{ js_bundle('about-transparency') }}
 {% endblock %}
 
 {% block content %}
 <main role="main">
   <header class="c-page-header">
     <div class="mzp-l-content mzp-t-content-lg">
-      <h1>{{ _('Transparency') }}</h1>
+      <h1>Transparency</h1>
     </div>
   </header>
 
   <div class="mzp-l-content mzp-t-content-lg">
     <p>
-      {{ _('Transparency is a key part of how Mozilla approaches user trust.') }}
-      {{ _('As an open source project that relies on open development, we build transparency into the way we write our code.') }}
-      {{ _('Additionally, our product documentation and notices describe how our products work and how we handle user data.') }}
+      Transparency is a key part of how Mozilla approaches user trust.
+      As an open source project that relies on open development, we build transparency into the way we write our code.
+      Additionally, our product documentation and notices describe how our products work and how we handle user data.
```

Comment:
nit: Remove the line-breaks/white-space for these three lines. They aren't rendered in the output. 
```suggestion
      Transparency is a key part of how Mozilla approaches user trust. As an open source project that relies on open development, we build transparency into the way we write our code. Additionally, our product documentation and notices describe how our products work and how we handle user data.
```